client/css/style.css

```diff
@@ -1483,6 +1483,25 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	display: inline-block;
 }
 
+.whois {
+	display: -ms-grid; /* Edge 15- */
```

Comment:
Euch, can we add autoprefixer into webpack rather than adding these into the source?